Detectors/MUON/CMakeLists.txt

```diff
@@ -1 +1,18 @@
+# **************************************************************************
```

Comment:
@dberzano : What do we do with such copied copyright notices. Should we adjust them to the new one?

Reply:
Definitely. If you agree, I am changing the check to include `CMakeLists.txt` as well (we only check .cxx, .h & at the moment). OK?

Reply:
Dear @sawenzel and @dberzano ,
sorry I missed the point here. Should I modify the copyright notice using e.g. the one that I find in Framework/Core/CMakeLists.txt?

In addition, I did not add a copyright in:
Detectors/MUON/MID/CMakeLists.txt
Detectors/MUON/MID/Base/CMakeLists.txt
I'm actually not sure if the notice should be added to all CMakeLists.txt (it looks like it is not systematically added in the CMakeLists.txt files that are currently in O2).
What is the policy? Should it be added to all files, independently of the extension?

Thanks,
cheers,
Diego

Reply:
At the moment, we only care about `.cxx` and `.h`, as you can [see here](https://github.com/alisw/alidist/blob/master/o2checkcode.sh#L54-L76). So for the moment please take it out.